### ISSUE TITLE
pypi page calls it lÃ¶ve instead of löve

### ISSUE THREAD

**Issue by idbrii** [pfirsich/makelove#32]
The [0.0.11](https://pypi.org/project/makelove/0.0.11/) and 0.0.10 releases on pypy describe makelove as:

> A packaging tool for [lÃ¶ve](https://love2d.org/) games

But [0.0.9](https://pypi.org/project/makelove/0.0.9/) has correct unicode:

> A packaging tool for [löve](https://love2d.org/) games

Doesn't look like the readme changed, so maybe a bug in the process of uploading to pypi? Or the environment it's done' in. (Not sure if it's manual copy/paste or if they automatically pull out the readme.)

**Comment by josh-perry** [comment]
Yeah I noticed that too. It looks scrambled in the .whl metadata file too so it's not pypi's fault.

![image](https://user-images.githubusercontent.com/6598916/222897265-16eaa74f-7a52-4e65-a151-6fc4935c6b84.png)

...but we haven't changed anything around that since 0.0.9 as far as I can see.
It comes from here by the way https://github.com/pfirsich/makelove/blob/master/setup.py#L4.

**Comment by idbrii** [comment]
That code should probably specify the encoding:

```
with open("README.md", "r", "utf-8") as fh:
    long_description = fh.read()
```

The [docs for `open`](https://docs.python.org/3/library/functions.html#open) say encoding is platform-dependent:

> In text mode, if encoding is not specified the encoding used is platform-dependent: [locale.getencoding()](https://docs.python.org/3/library/locale.html#locale.getencoding) is called to get the current locale encoding.

So maybe some platform or python installation change resulted in a bad/different default.

**Comment by pfirsich** [comment]
This probably broke, because on linux the default encoding for `open` is `utf-8`. Will be fixed for 0.0.12 (I hope :)). Thank you very much!
